pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -671,6 +700,55 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	return deployment, nil
 }
 
+func (c *Cluster) createOSDService(osd OSDInfo, labels map[string]string) (string, error) {
+	selectorLabels := map[string]string{
+		k8sutil.AppAttr: AppName,
+		"ceph-osd-id":   strconv.FormatInt(int64(osd.ID), 10),
+	}
+	svcDef := &v1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("rook-ceph-osd-%d", osd.ID),
+			Namespace: c.clusterInfo.Namespace,
+			Labels:    labels,
+		},
+		Spec: v1.ServiceSpec{
+			Selector: selectorLabels,
+			Ports: []v1.ServicePort{
+				{
+					Name:       "osd-port",
```

Comment:
nit: If we're naming the other one 2, how about naming this one 1 for consistency?
```suggestion
					Name:       "osd-port-1",
```